tls/tls_srv.c

```diff
@@ -62,7 +68,9 @@ ttls_parse_servername_ext(TlsCtx *tls, const unsigned char *buf, size_t len)
 	while (servername_list_size > 0) {
 		hostname_len = ((p[1] << 8) | p[2]);
 		if (hostname_len + 3 > servername_list_size) {
-			T_DBG("ClientHello: bad hostname size\n");
+			T_DBG("ClientHello: bad hostname size"
+			      " (%lu, expected not more %lu)\n",
+			      hostname_len, servername_list_size + 3);
```

Comment:
```suggestion
			      hostname_len, max_t(size_t, servername_list_size, 3) - 3);
```
we can't just subtract 3 because the variables are unsigned.